### PR TITLE
New package: python-gitlab-0.18

### PR DESCRIPTION
**Pull request by lafrenierejm**

### CODE REVIEW

File: srcpkgs/python-gitlab/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+homepage="https://github.com/gpocentek/python-gitlab"
+license="GPL-3"
+distfiles="${PYPI_SITE}/p/python-gitlab/python-gitlab-${version}.tar.gz"
+checksum=d60d67c82fedd8c3e4f0bb8b5241bf2df32307c98fdf2f02a94850e21db2d804
+alternatives="gitlab:gitlab:/usr/bin/gitlab2"
+
+python3-gitlab_package() {
+	noarch=yes
+	depends="python3-requests python3-six"
+	pycompile_module="github3"
```
Comment (Duncaen) [inline srcpkgs/python-gitlab/template]: typo

File: srcpkgs/python-gitlab/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'python-gitlab'
+pkgname=python-gitlab
+version=0.18
+revision=1
+noarch=yes
+wrksrc="${pkgname}-${version}"
```
Comment (Vaelatern) [inline srcpkgs/python-gitlab/template]: Totally redundant.

File: srcpkgs/python-gitlab/template [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+distfiles="${PYPI_SITE}/p/python-gitlab/python-gitlab-${version}.tar.gz"
+checksum=d60d67c82fedd8c3e4f0bb8b5241bf2df32307c98fdf2f02a94850e21db2d804
+alternatives="gitlab:gitlab:/usr/bin/gitlab2"
+
+python3-gitlab_package() {
+	noarch=yes
+	depends="python3-requests python3-six"
+	pycompile_module="gitlab"
+	short_desc="${short_desc/Python2/Python3}"
+	alternatives="gitlab:gitlab:/usr/bin/gitlab3"
```
Comment (Vaelatern) [inline srcpkgs/python-gitlab/template]: Can we name the alternatives group something like `python-gitlab`

File: srcpkgs/python-gitlab/template [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+revision=1
+noarch=yes
+build_style=python-module
+pycompile_module="gitlab"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python-requests python-six"
+short_desc="Wrapper for GitLab's API (Python2)"
+maintainer="Joseph LaFreniere <joseph@lafreniere.xyz>"
+homepage="https://github.com/gpocentek/python-gitlab"
+license="GPL-3"
```
Comment (asergi) [inline srcpkgs/python-gitlab/template]: `license="LGPL-3"`

File: srcpkgs/python-gitlab/template [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+pycompile_module="gitlab"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python-requests python-six"
+short_desc="Wrapper for GitLab's API (Python2)"
+maintainer="Joseph LaFreniere <joseph@lafreniere.xyz>"
+homepage="https://github.com/gpocentek/python-gitlab"
+license="LGPL-3"
+distfiles="${PYPI_SITE}/p/python-gitlab/python-gitlab-${version}.tar.gz"
+checksum=d60d67c82fedd8c3e4f0bb8b5241bf2df32307c98fdf2f02a94850e21db2d804
+alternatives="python-gitlab:python-gitlab:/usr/bin/gitlab2"
```
Comment (Vaelatern) [inline srcpkgs/python-gitlab/template]: `${group}:${a}:${b}` where `ln -s ${b} ${a}`
So `python-gitlab:gitlab:/usr/bin/gitlab2`

File: srcpkgs/python-gitlab/template [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+python3-gitlab_package() {
+	noarch=yes
+	depends="python3-requests python3-six"
+	pycompile_module="gitlab"
+	short_desc="${short_desc/Python2/Python3}"
+	alternatives="python-gitlab:python-gitlab:/usr/bin/gitlab3"
```
Comment (Vaelatern) [inline srcpkgs/python-gitlab/template]: And here

**Pull request merged**